### DISCUSSION TITLE
Is the website down?

### DISCUSSION THREAD

**Discussion by CyberShadow** [badges/shields#7609] [General] [answered]
I'm seeing a 503 "Application Error" page on https://shields.io .

**Comment by talwat** [comment]
Me too, I think so.

**Reply by Wjplatformer** [reply]
### It has currently been fixed.
 I'm not sure, though, if there is a GitHub pages site for shields. A separate website having a short explanation on the syntax of shields on a  GitHub page would be useful when main website is down or being changed as people like me have to constantly refer to the website for the syntax.

**Comment by PyvesB** [comment, marked as answer]
Hello 👋🏻 

I believe this was simply a temporary wobble due to some of our new host experiments. See the following comment for more details: https://github.com/badges/shields/issues/7574#issuecomment-1030637680
